lib/svg-to-vectordrawable.js

```diff
@@ -212,10 +212,6 @@ JS2XML.prototype.refactorData = function(data, floatPrecision) {
                 });
                 elem.removeAttr('transform');
             }
-            else {
```

Comment:
Do not understand why this _else_ needed? It extracts group content. But why do we need extracting?